response_operations_ui/views/messages.py

```diff
@@ -121,14 +121,46 @@ def view_messages():
         return render_template("messages.html", breadcrumbs=breadcrumbs, response_error=True)
 
 
+@messages_bp.route('/threads/<thread_id>', methods=['GET'])
+@login_required
+def view_conversation(thread_id):
+    try:
+        thread_conversation = message_controllers.get_conversation(thread_id)['messages']
+        refined_thread = [_refine(thread) for thread in reversed(thread_conversation)]
```

Comment:
Isn't thread here actually message, so should be `[_refine(message) for message in reversed(thread_conversation)]`?

Reply:
Rather than reverse the conversation, I think we should change the API to return it in the correct order for the UI

Reply:
@warrenbailey I agree, but should we not put down as tech debt external to this story. For the amount messages passed, has not memory or speed downside at the moment.